content/en/tracing/configure_data_security/_index.md

```diff
@@ -13,31 +13,129 @@ aliases:
+
+These categories are collected by the following language Tracing Libraries:
+
+{{% tabs %}}
+
+{{% tab ".NET" %}}
+
+| Category            | How the data is collected      |
+|---------------------|--------------|
+| Name                | Collected by default without obfuscation      |
+| Email               | Collected by default without obfuscation      |
+| Client IP           | Collected by default without obfuscation      |
+| Database statements | Collected by default with obfuscation      |
+| Geographic location | Never collected      |
+| URI parameters      | Collected by default with obfuscation      |
+| URI userinfo        | Never collected      |
+
+
+{{% /tab %}}
+
+{{% tab "Java" %}}
+
+| Category            | How the data is collected      |
+|---------------------|--------------|
+| Name                | Collected by default without obfuscation      |
+| Email               | Collected by default without obfuscation      |
+| Client IP           | Not collected by default, enabled with obfuscation     |
+| Database statements | Collected by default without obfuscation      |
+| Geographic location | Never collected      |
+| URI parameters      | Collected by default with obfuscation      |
+| URI userinfo        | Never collected      |
+
+{{% /tab %}}
+
+{{% tab "Node.js" %}}
+
+| Category            | How the data is collected      |
+|---------------------|--------------|
+| Name                | Collected by default without obfuscation      |
+| Email               | Collected by default without obfuscation      |
+| Client IP           | Collected by default without obfuscation     |
+| Database statements | Collected by default without obfuscation      |
+| Geographic location | Never collected      |
+| URI parameters      | Not collected by default, enabled with obfuscation      |
+| URI userinfo        | Never collected      |
+
+{{% /tab %}}
+
+{{% tab "PHP" %}}
+
+| Category            | How the data is collected      |
+|---------------------|--------------|
+| Name                | Never collected      |
+| Email               | Never collected      |
+| Client IP           | Never collected     |
+| Database statements | Collected by default with obfuscation      |
+| Geographic location | Never collected      |
+| URI parameters      | Collected by default with obfuscation      |
+| URI userinfo        | Collected by default with obfuscation      |
+
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+| Category            | How the data is collected      |
+|---------------------|--------------|
+| Name                | Collected by default without obfuscation      |
+| Email               | Collected by default without obfuscation      |
+| Client IP           | Not collected by default, enabled without obfuscation     |
+| Database statements | Collected by default with obfuscation      |
+| Geographic location | Collected by default without obfuscation      |
```

Comment:
I was not sure how to specify here that this location is only collected by a single integration (algoliasearch queries can contain coordinates).

@kayayarai Is there a pattern we can use here to add a note for further information?

Reply:
How about:
```suggestion
| Geographic location | Collected by default without obfuscation (by the [algoliasearch integration][1] only)      |
```